core/components/_helpers/margin.js

```diff
@@ -0,0 +1,33 @@
+import { spacing } from '@auth0/cosmos-tokens'
+
+const directions = ['top', 'bottom', 'left', 'right']
```

Comment:
I agree with @andresgalante, we shouldn't allow top margin.

Reply:
I'm not sure if we can do that right now. See thread: https://github.com/auth0/cosmos/pull/1059#issuecomment-435839056